MeetupPlatformApi/DataTransferObjects/MeetupInputDto.cs

```diff
@@ -1,13 +1,22 @@
-﻿namespace MeetupPlatformApi.DataTransferObjects;
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace MeetupPlatformApi.DataTransferObjects;
 
 public class MeetupInputDto
 {
+    [Required]
+    [MaxLength(30)]
+    [Display(Name = "Name of meetup")]
```

Comment:
Please, do not override field names. Api will be either used by other api or by client application (web ui, for example). In both cases these field names will be processed by an application and will not be exposed to the end user. Providing human-friendly names will only make it harder to parse these messages.